src/util/irep_ids.h

```diff
@@ -16,6 +16,8 @@ Author: Reuben Thomas, reuben.thomas@me.com
 
```

Comment:
To avoid the rather ugly perl in the travis config change that is part of this PR, is it worth considering tweaking the `#define USE_DSTRING` here to do something like:
```
#ifndef NO_USE_DSTRING
#define USE_DSTRING
#endif
```

Then the travis builds could just add `-DNO_USE_DSTRING` to the CXX_FLAGS?

Reply:
I have now added `USE_STD_STRING` to guard `USE_STRING` in the way you suggested, and thus also got rid of the ugly perl hack.